src/_langs/en/updates/2015-08-10-easily-duplicate-dom-nodes.markdown

```diff
@@ -0,0 +1,44 @@
+---
+rss: false
+layout: update
+published: true
+title: Easily duplicate DOM nodes
+date: 2015-08-10
+article:
+  written_on: 2015-08-10
+  updated_on: 2015-08-10
+authors:
+- umarhansa
+collection: updates
+type: tip
+category: tools
+product: chrome-devtools
+featured-image: /web/updates/images/2015-08-10-duplicate-dom-nodes/gplus-8f4a2ba4.png
+source_name: DevTips
+source_url: https://umaar.com/dev-tips/58-duplicate-dom/
+teaserblocks:
+- heading: ""
+  description: ""
+  image: ""
+- heading: ""
+  description: ""
+  image: ""
+- heading: ""
+  description: ""
+  image: ""
+permalink: /updates/2015/08/10/easily-duplicate-dom-nodes.html
+---
+<img src="/web/updates/images/2015-08-10-duplicate-dom-nodes/duplicate-dom.gif" alt="Duplicate DOM nodes">
+
+<em>Or what are essentially cut/copy &amp; paste operations.</em>
+
+You can easily change the DOM without having to edit the HTML as a big piece of string.

```

Comment:
Replace 'as a big piece of string' with 'a giant string'. (The former sounds odd to native speaker ears.)